### ISSUE TITLE
Textual changes in confirmation mail.

### ISSUE THREAD

**Issue by philip-n** [inbaz/ers#143]
Regarding the confirmation mail for bank transfer payment pasted below.

Note: As few people usually make most of the fuss, I tend towards rather strict formulations. Less "please", more "Do it like this. Otherwise it won't work." :-)

[1] There are multiple, partially redundant statements about payment method: Line 4, line 6, line 7, line 10. If bank transfer was chosen, line 6 and 10 should be left out.

[2] I'd suggest a text change for line 5: "To complete your order, we need to receive your bank transfer within 14 days. If you ordered a discounted ticket (phase 1 or phase 2), please make sure that the payment is processed on time (31st of May / 15th of June)."

[3] Line 10: The resume link is no there (because payment was bank transfer?). Leave out whole line on bank transfer? See [1].

[4] Line 8 and line 9 are duplicates of each other. Replace by the following?
"Please make sure to state your order ID CAD15G73 as reference at your transfer form. Otherwise we won't be able to match your payment and your order can't be processed."

[5] Line 11: The Terms and agreement are not yet attached to the mail.

[6] There is no link to either the EJC homepage or the ERS in the mail.
a) Change line 13 to "We are looking forward to welcoming you in Bruneck! Please check our homepage www.ejc2015.org once in a while for the latest information about EJC 2015."
b) Paste the text snippet from #142 (with according links) at the bottom of the mail, after the liste of all purchased goods?

---

```
1 Dear ,

2 This is the confirmation for your order from 10.03.2015 with the order ID CAD15G73.

3 Thank you for registering for the 38th European Juggling Convention! Your registration helps us to improve the EJC 2015 for you and for all fellow jugglers.

4 You chose Bank Bruneck as payment method.

5 Please transfer the entire amount of your order within 14 days. To gain from the pre-booking conditions, please make sure that the payment is processed on time (31st of May / 15th of June), even if there are less than 14 days left.

6 In case you chose 'bank transfer' as payment method please transfer the entire amount of your order to the following account:

7 Please transfer the amount by SEPA bank transfer to the following account:

8 To make the orgas life easy, please add the reason for payment: CAD15G73
Owner 	STICHTING EUROPEAN JUGGLING ASSOCIATION
IBAN 	NL84 INGB 0007 8721 92
BIC 	INGBNL2A
Amount 	15,00 €
reason for payment 	CAD15G73

9 Please state your order ID CAD15G73 as reference at your transfer form.

10 In case you chose credit card as payment method and your payment was not successful, you can restart your credit card payment via the following link:
%resume_link%

11 Please consider our general terms and conditions, which are attached to this email. To be able to read the document, you need to install Adobe Acrobat Reader, which you can download here: http://get.adobe.com/de/reader

12 Your eTickets and further information concerning the 38th European Juggling Convention will be delivered in another email after your payment has been confirmed.

13 We are looking forward to welcoming you in Bruneck!

14 Best regards,
Your European Juggling Association and the local EJC 2015 organizers

15 You purchased the following items:
unassigned items
Product 	Amount 	Price 	
week ticket youth 	1 	5,00 € 	
day ticket youth ( day: August 8 ) 	1 	10,00 € 	
	Subtotal: 	15,00 € 	
Order amount: 	15,00 €
Payment Charge: 	0,00 €
Total amount: 	15,00 €
```

**Comment by inbaz** [comment]
Many cool ideas. This was just a copy&paste of the 2011 mail. Should we add a header and a footer to this mail, then we're able to add relevant infos like link to the homepage to the footer.

**Comment by smadamit** [comment]
This is a sub-issue of #192

**Comment by inbaz** [comment]
May you want to get the actual confirmation mail for bank transfer and credit card payment from the system and look through it? there have been made some changes, maybe that fixes some of the changes.

**Comment by philip-n** [comment]
Just created a new emails_pn document on the drive with some additions to the prospected mails.

@inbaz @smadamit @jackcaveman

**Comment by philip-n** [comment]
Didn't seem to get through. Edited the email document instead.

**Comment by inbaz** [comment]
@philip-n Can again have a look it the document is really in the ers folder of the drive?

**Comment by philip-n** [comment]
Sorry - I had mailed them to you instead (html versions). Just resent the mail - let me know if you didn't receive it or if it is unclear!

**Comment by philip-n** [comment]
@inbaz : In case the new confirmation mail etc. won't be adopted (which I guess), there are two to do's for the current version:
* replace "(15th of May / 15th of June)" by "(15th of May / 15th of June)" as pbank transfer deadline in the "Please transfer the eintire amount..." paragraph ( @smadamit : correct?)
* remove a stray ">" in the line after the link to the order id:
```
To see the status of your payment you can visit the following link: Your order details
>

Please consider our general terms and conditions, which are attached to this email. To be able to read the document, you need to install Adobe Acrobat Reader, which you can download here: http://get.adobe.com/reader

**Comment by inbaz** [comment]
Changed email in 0.2.68, please test bank transfer and credit card payment orders and read through the complete mail if there are any typos.

Hint: The language selector is not in the mail, yet.

**Comment by smadamit** [comment]
@philip-n Can you do that since you have a better overview to this issue.

**Comment by philip-n** [comment]
On my way.

**Comment by philip-n** [comment]
@smadamit : When wil tickets be sent? When payment is received or on a later date? Mail currently says "When payment is received", but I think I have read somewhere that it will be in July (?).

**Comment by smadamit** [comment]
In general "When payment is received". But the feature is not implemented yet. So it is not clear at the moment when the feature is ready. When it's ready all paid etickets will be sent by in one step. After that: "When payment is received".

**Comment by smadamit** [comment]
Looks much better now. Two issues from my point of view for the bank transfer mail:
- skip the first line:
 - Please transfer the entire amount of your order to the following account:
 - Please transfer the amount by SEPA bank transfer to the following account:
- add one break line between the two lines:
 - Total amount:	0,12 €
 - This mail was sent to you by the EJC Registration System (ERS), operated by the European Juggling Association (EJA) on behalf of EJC 2015.

**Comment by philip-n** [comment]
Confirmation mail looks nice! Also, green top bar is much friendlier than the dark one :-)

Couple of minor corrections (sorry!):
1. It's the "**3**8th European Juggling Convention" (not the **2**8th)
2. In the part with the bank transfer, it says:
```
Please transfer the entire amount of your order to the following account:

Please transfer the amount by SEPA bank transfer to the following account:
```
Line 1 or 2 should be removed (doesn't matter which).
3. Personally, I'd skip the line 
```
To make the orgas life easy, please add the reason for payment: NXW6X851
```
or rephrase it to "To make the organizer's life easier, please add ..."

**Comment by philip-n** [comment]
One more thing which I'll add as another issue, just a mom.

**Comment by smadamit** [comment]
4 times: the **28th** European Juggling Convention -> the **38th** European Juggling Convention

**Comment by inbaz** [comment]
That was a variable. :) fixed in 0.2.71

**Comment by smadamit** [comment]
Ah OK :-)

Please have a look to the last comment of philip. Some more corrections.

**Comment by inbaz** [comment]
fixed in 0.3.5

**Comment by smadamit** [comment]
works for me for going live.

@philip-n Please recheck in SecondLife. 1. and 2. is fixed. 3. and 4. ->I'm not sure.

**Comment by philip-n** [comment]
3. is solved.
4. also solved (see below).

```
4. Pure cosmetics: Change "You purchased the following items:" to "You purchased the following items (ordered by person):" (currently it says "... following items" and what follows is the name of a person)
```
Ooops. Totally ambiguous. What Andi implemented is: "You purchased the following items (ordered by %purchaser%):". What I meant was literally "You purchased the following items (ordered by person):" in the sense of *The items are grouped by the persons they belong to*. But I actually like Andis version better :-)

**Comment by inbaz** [comment]
bump, is this issue completely solved? If so, please close.

**Comment by smadamit** [comment]
I don't know. Someone have to check it. @philip-n @PiQuer or I have to do it...

**Comment by smadamit** [comment]
@inbaz Just one small text change. From:

**To make the orgas life easy, please add the reason for payment: NXW6X851**

to:

**To make the organizer's life easier, please add the reason for payment: NXW6X851**

Rest of the ticket is solved, so just close it after this small fix.

**Comment by inbaz** [comment]
fixed with a small change from a random english teacher. :) organisers' because it's plural.
